pkg/trace/api/api.go

```diff
@@ -105,19 +105,22 @@ func NewHTTPReceiver(conf *config.AgentConfig, dynConf *sampler.DynamicConfig, o
 func (r *HTTPReceiver) buildMux() *http.ServeMux {
 	mux := http.NewServeMux()
 
+	hash, infoHandler := r.makeInfoHandler()
 	r.attachDebugHandlers(mux)
 	for _, e := range endpoints {
-		mux.Handle(e.Pattern, replyWithVersion(e.Handler(r)))
+		mux.Handle(e.Pattern, replyWithVersion(hash, e.Handler(r)))
 	}
-	mux.HandleFunc("/info", r.makeInfoHandler())
+	mux.HandleFunc("/info", infoHandler)
```

Comment:
The /info endpoint need to reply with the hash for the change of version detection.

Reply:
Done.